Extension/src/common.ts

```diff
@@ -806,11 +803,12 @@ export function promptForReloadWindowDueToSettingsChange(): void {
 
 export function promptReloadWindow(message: string): void {
     const reload: string = localize("reload.string", "Reload");
-    vscode.window.showInformationMessage(message, reload).then((value?: string) => {
+    (async () => { 
```

Comment:
I'd avoid using a async lambda to wrapper awaited calls.  If you don't also await the lamda, it's not really accomplishing anything.  If promptReloadWindow cannot be made async, this will need to continue to use `then`.